test/kwokctl/kwokctl_restart_test.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DIR="$(dirname "${BASH_SOURCE[0]}")"
+
+DIR="$(realpath "${DIR}")"
+
+RELEASES=()
+
+function usage() {
+  echo "Usage: $0 <kube-version...>"
+  echo "  <kube-version> is the version of kubernetes to test against."
+}
+
+function args() {
+  if [[ $# -eq 0 ]]; then
+    usage
+    exit 1
+  fi
+  while [[ $# -gt 0 ]]; do
+    RELEASES+=("${1}")
+    shift
+  done
+}
+
+function test_create_cluster() {
+  local release="${1}"
+  local name="${2}"
+
+  KWOK_KUBE_VERSION="${release}" kwokctl create cluster --name "${name}" --quiet-pull
```

Comment:
https://github.com/kubernetes-sigs/kwok/blob/1498f7709262796f52520481868c5a687c19d184/test/kwokctl/kwokctl_workable_test.sh#L69

Referring to it, enabling Prometheus in the test will not work after restart if it is modified in the current way, because its port is lost